python/taichi/misc/gui.py

```diff
@@ -155,10 +195,33 @@ def get_key_event(self):
             self.key_pressed.add(e.key)
         else:
             self.key_pressed.discard(e.key)
+        return e
+
+    def _get_key_event(self):  # ABi (assume has_key_event)
+        e = self._peek_key_event()
         self.core.pop_key_event_head()
         return e
 
-    def is_pressed(self, *keys):
+    def wait_key_event(self):  # ABi
+        self.core.wait_key_event()
+
+    def peek_key_event(self):  # Unique APi
+        e = None
+        if self.has_key_event():
+            e = self._peek_key_event()
+        return e
+
+    def get_key_event(self):  # Unique APi
```

Comment:
Again, if `APi` stands for *application programming interfaces*, why is `i` lowercased?

Reply:
For fun :)